raw_data/preprocessed/africa/liberia_english.txt

```diff
@@ -0,0 +1,376 @@
+
+Mitigation Plan, Audits and Monitoring for Existing Industries, Projects, Activities
+(27) [1] The owner/management of all industries, projects, activities existing or initiated prior to the effective date of this Law, but which fall into the category of activities subject to an environmental impact assessment, shall within days of the effective date, submit to the Agency Plans for improving environmental performance, which shall include: [a] Identification of the major environmental effects; and [b] A comprehensive mitigation plan in accordance with section 15 of this Law. [2] The mitigation plan shall be reviewed by the Agency in consultation with the Line Ministry, an agreement on mitigation strategies and timeframe reached with the owner/management. [3] Thereafter, all such industries, projects and activities are subject to periodic audits and ongoing monitoring in accordance with section 24, 25 and 26 of this Law. [4] Any owner/management of any industries, project, activities existing or initiated prior to the effective date of this Law, who fails to comply with the directions issued under section 27 commits an offence and is liable, on conviction, to a fine not exceeding 10,000 US DOLLARS or imprisonment for a period not exceeding 5 years, or to both.
+
+Transfer of Environmental Impact Assessment License
+(28) [1] The holder of may transfer an environmental impact assessment license to another person only in respect of the project/activity in relation to which the license was issued; [2] Where the license referred to in subsection [1] is transferred, the person to whom it was issued must give the Agency prior notice of the pending transfer and both the license and the person to whom it is transferred to, shall jointly and in writing notify the Agency of the transfer, within days of the transfer; [3] The holder of the license shall, where no joint notification of a transfer is given be deemed for the purposes of this Law to be the owner or the person having charge, or management or control of the project; [4] A transfer of an environmental impact assessment license shall be effective from the date the agency is notified of the transfer; [5] The Agency may prescribe additional procedures for transference of environmental impact license; [6] Any person who contravenes the provisions of this section commits an offence and is liable on conviction to a fine not exceeding 25,000 US DOLLARS or to imprisonment for a period not exceeding 10 years or to both.
```

Comment:
Please replace:

* `The holder of may` with `The holder may`
* `issued; [2]` with `issued. [2]`
* `both the license and the person` with `both the licensee and the person`
* `transfer; [3]` with `transfer. [3]`
* `project; [4]` with `project. [4]`
* `transfer; [5]` with `transfer. [5]`
* `license; [6]` with `license. [6]`

I will resume my review this afternoon.